pkg/collector/py/py3_checker_nix.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build cpython,!windows
+
+package py
+
+import (
+	"path/filepath"
+)
+
+const (
+	// Waiting for final name
+	py3LinterBin = "a7_validate"
```

Comment:
Do we know the name yet?

Reply:
yes, that's the final name. Forgot to remove the comment !